package.json

```diff
@@ -13,7 +13,7 @@
         "cli"
     ],
     "scripts": {
-        "configure:git": "echo Ensuring git hooksPath is set to .githooks directory && git config core.hooksPath .githooks && chmod +x .githooks/*",
+        "configure:git": "echo Ensuring git hooksPath is set to .githooks directory; git config core.hooksPath .githooks; chmod +x .githooks/*",
```

Comment:
Is the intention here to run the 3rd command even when the 2nd fails?

Reply:
Yes - realized after I pushed this originally that this will run on any install, not just a local project setup. So if you're attempting to install it globally, there's no `.githooks` directory - using the `&&` operators will fail the install if any of the commands return a non-zero exit code